fence/config.py

```
@@ -150,28 +152,37 @@ def post_process(self):
                 logger.warning(
                     f"IdP '{idp_id}' is using multifactor_auth_claim_info '{mfa_info['claim']}', which is neither AMR or ACR. Unable to determine if a user used MFA. Fence will continue and assume they have not used MFA."
                 )
+        dbgap_configs = self._configs["dbGaP"]
+        if isinstance(dbgap_configs, list):
+            corrected_dbgap_configs = dbgap_configs
+        else:  # assume is a dict
```

Instead of assuming, why not actually check if `dbgap_configs` is a dict and raise an exception otherwise

---

Hmm, yeah I thought about that initially but decided to just minimize the changes. If it were ever not a dict though, that would be useful information. I'll update it!